contracts/strategies/DonationVotingOffchain.sol

```diff
@@ -197,12 +201,16 @@ contract DonationVotingOffchain is CoreBaseStrategy, RecipientsExtension {
         _updatePoolTimestamps(_registrationStartTime, _registrationEndTime);
     }
 
-    /// @notice Claim allocated tokens
-    /// @param _claims Claims to be claimed
-    function claimAllocation(Claim[] calldata _claims) external onlyAfterAllocation {
+    /// @param _data The data to be decoded
```

Comment:
missing notice, and would be great to add dev comments on this and all other affected functions on how directTransfer is being used, e.g. this function should not be called if directTransfer is true ...